google/cloud/security/inventory/pipelines/load_appengine_pipeline.py

```diff
@@ -78,7 +128,16 @@ def _transform(self, resource_from_api):
 
     def run(self):
         """Run the pipeline."""
-        apps = self._retrieve()
-        loadable_apps = self._transform(apps)
-        self._load(self.RESOURCE_NAME, loadable_apps)
+        apps, loadable_services, loadable_versions, loadable_instances = (
+            self._retrieve())
+
+        if apps:
+            loadable_apps = self._transform(apps)
+            self._load(self.RESOURCE_NAME, loadable_apps)
         self._get_loaded_count()
+
+        # TODO: Make _get_loaded_count() support multiple resources in a single
```

Comment:
Nit: You could put this under the if apps: since if there are no apps there won't be any other resources either.

Reply:
Done!